config/routes.rb

```diff
@@ -1,10 +1,27 @@
 Rails.application.routes.draw do
+  get 'foods/index'
+  devise_for :users
+  get 'recipes/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
+  get 'up' => 'rails/health#show', as: :rails_health_check
 
   # Defines the root path route ("/")
   # root "posts#index"
```

Comment:
- [ ] Please kindly remove this commented code in order to make your work neater. ✔️